libyul/backends/wasm/WasmDialect.cpp

```diff
@@ -56,10 +56,17 @@ WasmDialect::WasmDialect():
 	addFunction("i64.eqz", 1, 1);
 	m_functions["i64.eqz"_yulstring].returns.front() = "i32"_yulstring;
 
+	addFunction("i64.clz", 1, 1);
+	m_functions["i64.clz"_yulstring].returns.front() = "i64"_yulstring;
```

Comment:
Are you sure this returns `i64`? If it does no need to state it (since that's by default).

Reply:
According to https://github.com/sunfishcode/wasm-reference-manual/blob/master/WebAssembly.md#integer-count-leading-zeros
it returns `i64`. If it's by default I will remove it.

Reply:
It is by default.